### DISCUSSION TITLE
mmseg库整理各模型的整体对比结果

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#619] [MMSegmentation]
# mmseg库整理各模型的整体对比结果 

整理各个模型整体的对比结果，包括效果和速度，类似[https://github.com/AI-Tianlong/Useful-Tools/assets/50650583/fd31e472-7285-41ad-a567-a7721e74f9fb](https://github.com/AI-Tianlong/Useful-Tools/assets/50650583/fd31e472-7285-41ad-a567-a7721e74f9fb) 

## Task Information

**Repository**：MMSegmentation

**Difficulty**：Easy

**Points**：20

**Estimated Time Consumption**：14 days

## Task Status

**Claimant's Github ID**：None

**Claim Time**：None

**PR Link**：None

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by ooooo-create** [comment]
Claim the mission
